server/ladder_service.py

```diff
@@ -50,46 +50,18 @@ def __init__(
         self._db = database
         self._informed_players: Set[Player] = set()
         self.game_service = game_service
-
-        # Fallback legacy map pool and matchmaker queue
-        self.ladder_1v1_map_pool = MapPool(0, "ladder1v1")
-        self.queues = {
-            "ladder1v1": MatchmakerQueue(
-                game_service,
-                name="ladder1v1",
-                featured_mod=FeaturedModType.LADDER_1V1,
-                rating_type=RatingType.LADDER_1V1,
-                map_pools=[(self.ladder_1v1_map_pool, None, None)]
-            )
-        }
+        self.queues = {}
 
         self._searches: Dict[Player, Dict[str, Search]] = defaultdict(dict)
 
     async def initialize(self) -> None:
-        # TODO: Starting hardcoded queues here
-        for queue in self.queues.values():
-            queue.initialize()
-
         await self.update_data()
         self._update_cron = aiocron.crontab("*/10 * * * *", func=self.update_data)
 
         self.start_queue_handlers()
```

Comment:
This function needs to go as well. Right now each queue ends up getting two handlers trying to pull from it at the same time.